CMake/add_project_fletch.cmake

```diff
@@ -62,10 +62,11 @@ ExternalProject_Add(fletch
     -Dfletch_ENABLE_Qt:BOOL=OFF
     -Dfletch_ENABLE_Snappy:BOOL=${CAFFE_SUPPORT}
     -Dfletch_ENABLE_SuiteSparse:BOOL=OFF
-    -Dfletch_ENABLE_TinyXML:BOOL=ON
+    -Dfletch_ENABLE_TinyXML1:BOOL=ON
     -Dfletch_ENABLE_VTK:BOOL=OFF
     -Dfletch_ENABLE_VXL:BOOL=ON
     -Dfletch_ENABLE_YAMLcpp:BOOL=ON
+    -DYAML_CPP_BUILD_WITH_FPIC:BOOL=ON
```

Comment:
Based on the purpose of fletch, I don't see a use-case where this flag would be turned off. Shared libraries are build with fPIC and static libraries that are being used for another project would require the fPIC.

Reply:
Ah, for whatever reason I had to explicitly tell YAML_CPP in fletch to build with FPIC.  Based on your comment then, I guess it would make more sense to just remove the flag in fletch and always build YAML_CCP with FPIC?

Reply:
yes